topics/distlock.md

```diff
@@ -228,4 +228,4 @@ Thanks in advance!
 ## Analysis of Redlock
 ---
 
-1. Martin Kleppmann [analyzed Redlock here](https://martin.kleppmann.com/2016/02/08/how-to-do-distributed-locking.html). A counterpoint to this analysis can be [found here](http://antirez.com/news/101).
+1. Martin Kleppmann [analyzed Redlock here](https://martin.kleppmann.com/2016/02/08/how-to-do-distributed-locking.html). A counterpoint to this analysis can be [found here](https://web.archive.org/web/20241209045142/https://antirez.com/news/101).
```

Comment:
https in this one? It's http in all the others

Reply:
Weird.

I think what happened is that I just pasted the links into wayback machine and use the right arrow to get the latest. I guess this got the https version.

I'm glad to change it but I don't think it has any bearing on what's being served.